components/places/android/src/main/java/org/mozilla/places/PlacesConnection.kt

```diff
@@ -223,6 +237,25 @@ interface PlacesAPI {
      */
     fun getVisitedUrlsInRange(start: Long, end: Long = Long.MAX_VALUE, includeRemote: Boolean = true): List<String>
 
+    /**
+     * Deletes all information about the given URL. If the places has previously
+     * been synced, a tombstone will be written to the sync server, meaning
+     * the place should be deleted on all synced devices.
```

Comment:
Hmm, I feel like multiple places mapping to the same URL is universally a bug. Once that happens, one of them is basically immortal anyway (Unless it gets deduped?).

It would be nice to know how often this happens...